getting_hired/preparing_for_job_search/professional_networking.md

```diff
@@ -0,0 +1,80 @@
+
+* [Join the Odin community](https://www.theodinproject.com/lessons/foundations-join-the-odin-community).
+* [LinkedIn](https://www.linkedin.com/) - your profile acts like your virtual resume and first impressions are important. Put some time into [optimizing your profile](https://www.samanthaming.com/blog/tips-to-optimize-your-linkedin-profile-for-developers/). Then start following companies you would like to work at or find interesting, and individuals you can learn from. Engage with their content with questions and comments and consider creating your own content about your developer journey. Connect with people who engage with you and aim to make these connections genuine by looking at someone’s profile and finding something in common to talk about.
+* [X](https://twitter.com/) - this is a great platform to keep a finger on the pulse of a given domain. You can follow companies and individuals, engage with their posts, and create your own content.
+* [Discord](https://discord.com/) - there’s likely a discord channel dedicated to a given language or framework where you can ask questions and get help. You can sign up for Discord and search for what you are interested in e.g., NodeJS, Ruby on Rails, and request to join.
+* Open Source projects - when you have the skillset and confidence to contribute to open-source projects, you will be working directly with project maintainers and contributors. It is a great way to connect with other developers while working towards a common good.
+
+### Build Genuine Connections
+
+The economy of professional networking rewards authenticity and mutual respect. If you are going to ask a favor of someone, you are far more likely to get help if you have developed a genuine connection. That is to say, please do not approach people ‘cold’ and ask them out of the blue to connect with you, or for a favor like ‘help me get this job in your company’ or ‘please be my mentor’. This may be perceived as disrespectful. People are busy, and they reserve their limited time for helping those they know and respect.
+
+Your best strategy is to start conversations with a genuine interest in the person and connect on a mutual interest. For example, you could review someone’s profile on LinkedIn and find something in common, or begin a conversation in-person with questions related to what the person does, how they got to where they are, and what challenges they face in their daily work. Make it about them, not you. Show that you are interested in them as a person, not that you want them to give you something. Actively listen to what they say and ask follow-up questions and even offer to help with something if you are in a position to do so. Think of the interaction like dating, you don’t ask someone to be your partner in the first question (do you? lol), so the same applies here. If your initial conversation goes well, then you can follow up with a request to connect.
```

Comment:
I feel like this paragraph is hinting at informational interviews. Not suggesting we write an explainer on that from scratch (unless you want to) but linking to something that explains informational interviews would feel good here.

Reply:
Yeah good call. That adds value here.